defaults/main.yml

```diff
@@ -12,10 +12,10 @@ keep_only_specified: False
 nginx_installation_type: "packages"
 nginx_binary_name: "nginx"
 nginx_service_name: "{{nginx_binary_name}}"
-nginx_conf_dir: "{% if ansible_os_family == 'FreeBSD' %}/usr/local/etc/nginx{% else %}/etc/nginx{% endif %}"
+nginx_conf_dir: /etc/nginx
```

Comment:
nginx_conf_dir and nginx_user will still be overridable ?

Reply:
Yes. Anything defined in vars/ which will be evaluated will have precedence over the defaults.

Reply:
Ok thanks for the clarification.
I think it can be confusing if someone read the file.
Maybe we can do something that describe of the behavious like
```
nginx_conf_dir: "{{nginx_default_conf_dir}}"
nginx_user: "{{nginx_default_user}}"
```
With generic default in vars/empty.yml (empty.yml could become default.yml btw)
What do you think?

Reply:
This would somehow defeat the purpose of having this defaults configuration file, if the values are actually to be found in the `empty.yml`. I would rather document that for supported environments some values might be overwritten in separate files.

Reply:
In all case, for specific OS value have to be found in `vars/{{ ansible_os_family }}.yml`, I just wanted to do that generically.
A documentation is a good alternative too.
If you can implement either of the solutions, I will merge the PR after.